src/utils/time.hpp

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2024 Renesas Electronics Corporation.
+ * Copyright (C) 2024 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef TIME_HPP_
+#define TIME_HPP_
+
+#include <chrono>
+#include <optional>
+#include <string>
+
+#include <aos/common/tools/error.hpp>
+
+/***********************************************************************************************************************
```

Comment:
I wonder if we need to use namespace here. Like `time`. Need to discuss with team.

Reply:
use UtilsTime